plugins/org.jkiss.dbeaver.ext.generic.ui/src/org/jkiss/dbeaver/ext/generic/views/GenericConnectionPage.java

```diff
@@ -169,7 +169,13 @@ public void widgetSelected(SelectionEvent e) {
 
         {
             Label serverLabel = new Label(settingsGroup, SWT.NONE);
-            serverLabel.setText(GenericMessages.dialog_connection_server_label);
+            DBPDriver driver = site.getActiveDataSource().getDriver();
```

Comment:
The idea was to populate it thru bundle.properties, no need to add getPasswordFieldLabel and SQLiteMessages